### ISSUE TITLE
Porcupine Issue: Plugins: Failed to load 'pv_porcupine.dll' because one or more of its dependencies could not be loaded.

### ISSUE THREAD

**Issue by Mcnath** [Picovoice/porcupine#610]
### Expected behaviour

Porcupine initialized normally

try
        {
            _porcupineManager = PorcupineManager.FromKeywordPaths(accessKey,keywordPaths, OnWakeWordDetected);
            Debug.Log("Porcupine manager initialized");
        }

### Actual behaviour

Porcupine failed to initialize. I thought initially the problem was in my code so I try to run demo which throw the same error. I used to be able to test my code in editor runtime.

Plugins: Failed to load 'Assets/Porcupine/Plugins/windows/amd64/pv_porcupine.dll' because one or more of its dependencies could not be loaded.
Pv.Unity.Porcupine:FromKeywordPaths(String, IEnumerable`1, String, IEnumerable`1) (at Assets/Porcupine/Porcupine.cs:150)
Pv.Unity.PorcupineManager:FromKeywordPaths(String, IEnumerable`1, Action`1, String, IEnumerable`1, Action`1) (at Assets/Porcupine/PorcupineManager.cs:66)
PorcupineColorChangeCheckpoint:Start() (at Assets/Scripts/PorcupineColorChangeCheckpoint.cs:34) // this is part of my code that call PorcupineManager

### Steps to reproduce the behaviour

1. Followed tutorial guide for Unity API
2. Run the project in editor mode.

Notes: Not sure this played part or not but for some reason PorcupineException is not found in Vs editor. I also have created new blank project to test the package and encountered exact same issue.

Unity version used: 2019.3.8f1

**Comment by ksyeo1010** [comment]
Hi, I have few questions to ask to help you resolve your issue since I can't reproduce it on mine.

1. Which version of the package are you using? Make sure you use 2.0.1 the latest one.
2. Did you import all files in the package? Please check if the file `${PROJECT_ROOT}\Assets\Porcupine\Plugins\windows\amd64\pv_porcupine.dll` exists. If not, the package has not been imported properly. This might be the case since you mentioned PorcupineException is not found. Please re import the package if so.
3. Check if you have the following dlls in system32:
- C:\Windows\system32\advapi32.dll
- C:\Windows\system32\kerner32.dll
- C:\Windows\system32\msvcrt.dll
- C:\Windows\system32\ws2_32.dll

**Comment by Mcnath** [comment]
Thank you so much for you help.

1. I was using ver 2.0.0.
2. I did import most of the files minus the plugin for platform that I'm not using.
3. Yes, i have all the dlls in my system.

After posting, I switched over to ver 2.0.1 which seems to fix the issue. But Porcupine still not initialized, with the error now become "AccessKey activation error". I have already included accesskey into my code which i get form my account. But the scene still give the same error.
